docs/supported_models.md

```diff
@@ -1,4 +1,4 @@
-# Supported models (100 total)
+# Supported models (101 total)
```

Comment:
No need to modify supported_models next time. This will be done automatically by github action

Reply:
Makes sense 👍